assessments/projects/rook/self-assessment.md

```diff
@@ -0,0 +1,472 @@
+# Rook Self-assessment
+
+## Self-assessment outline
+
+## Table of contents
+
+* [Metadata](#metadata)
+  * [Security links](#security-links)
+* [Overview](#overview)
+  * [Actors](#actors)
+  * [Actions](#actions)
+  * [Background](#background)
+  * [Goals](#goals)
+  * [Non-goals](#non-goals)
+* [Self-assessment use](#self-assessment-use)
+* [Security functions and features](#security-functions-and-features)
+* [Project compliance](#project-compliance)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+* [Appendix](#appendix)
+
+## Metadata
+
+A table at the top for quick reference information, later used for indexing.
```

Comment:
```suggestion
```

Reply:
fixed this one manually above ([commit](https://github.com/dhauss/tag-security/commit/ab38ebc933b142aa6f432e7b7516f59a58975920)), thanks for catching this